Client/src/Features/Monitors/monitorsSlice.js

```diff
@@ -46,7 +46,14 @@ export const getMonitorsByUserId = createAsyncThunk(
 const monitorsSlice = createSlice({
   name: "monitors",
   initialState,
-  reducers: {},
+  reducers: {
+    clearMonitorState: (state) => {
+      state.isLoading = false;
+      state.monitors = [];
+      state.success = null;
+      state.msg = null;
```

Comment:
You could technically reset the monitor state to initialState in case we add more values, something like `clearMonitorState: () => initialState`, but I doubt it's needed.